src/models/base.js

```diff
@@ -0,0 +1,16 @@
+import Phaser from 'phaser'
+
+const {Geom} = Phaser
+const {Circle: PCircle, Triangle: PTriangle} = Geom
+
+// abstract over the original code we wrote and using proper PhaserCircles
+export const Circle = {
+  of: ({x, y, a, _radius}) => new PCircle(x, y, _radius || a)
+}
+
+export const Triangle = {
+  equilateral: (x, y, length) => {
```

Comment:
Use an object for args per everything else.

Reply:
Agreed - I wanna use objects to facilitate that FPO pattern later. I think once you implement it first a bit I'll adopt it more.